xwiki-rendering-macros/xwiki-rendering-macro-figure/src/main/java/org/xwiki/rendering/internal/macro/figure/FigureCaptionMacro.java

```diff
@@ -86,13 +87,16 @@ public boolean supportsInlineMode()
     {
         List<Block> result = Collections.emptyList();
 
-        // If we're not inside a FigureBlock then don't do anything.
-        Block parent = context.getCurrentMacroBlock().getParent();
-        if (parent instanceof FigureBlock) {
+        Block parentBlock = context.getCurrentMacroBlock().getParent();
+
+        // we don't want to do anything if we're not in a FigureBlock
```

Comment:
IMO this was not needed to be changed. And you put a lowercase for starting the sentence ;)

Reply:
Yes would probably be cleaner to go back to previous version since it does not looks like a obvious improvement.